.travis.yml

```diff
@@ -32,7 +33,9 @@ deploy:
   api_key:
     secure: nxaqYrkXLGL3W20/eCnf63DLjMrQAhEuW44jggh1/nI383goa+u6w0bBtWCxRdVzos7t4dpVfS6+kv6oIHacm9zVA+RYrqy5opzCJhq8lmXVVRijbALzUeiFif2HURMaKWj0ynRNVlAyBHzazPTLZVWywifpdSubSkuMWkl20cmuKu/Hg3c1EC9se3OYhhTHx3Hya7xSrctrDEYLsEBAUZzkKfscqRVqwwltS88CgIMtRISDpSBGrtH0t1uAH6NitTSguGgb+QEpqnELcRLymX2G1yzMA4Xr5c/L34MfbBKf8vIuG9t411xYuLoyKoUbroTWxSnPwlSy6PHz+QJ7UCXbDkATOGO3chxlKxglppvI/G3n2YP5Zf2dAaDlHblpvarh55i/4i4sKB2AbvvzkIHrQJwUgmLCbpN8/Vp9GWcGkd6i5U7F8tNInCs6ttX3oGvGOfYEXs02Ctyiea4LAqk4S7GZTuV2QXqxXglL4eRIwZ4UETiwgoAAtHma63Eq7+9t2ykMlk7zAK96FGwJrB97wa08aPuSxL94IYEBmn9Ht/vKXRiNQMvpnfp4rWQtL3cqbVyYAg5EjKb4PsBmnb91+RXtnWFOY1RpZGt8sPXYd+KZYzN1BXTFJEpaLLsIDN6r7nMcAvJDUmucaM+m7giPXz1ZBGAic3UBM1qMCgI=
   file_glob: true
-  file: game-core/build/artifacts/*
+  file: 
+    - game-core/build/artifacts/*
+    - lobby/build/distributions/lobby-$LOBBY_VERSION-server.zip
```

Comment:
I think it would be easier to copy artifacts from all subprojects to a folder in the root project.  A Gradle task in each subproject would be useful for that.  That is, each subproject should have a `prepareArtifacts` task that copies the publishable artifacts to _build/artifacts_.

Otherwise, you're going to have to modify the _.travis/generate_artifact_checksums_ script to account for the artifacts being spread out in multiple locations.

Reply:
I'm actually not 100% sure I agree; It's half of one or the other. Artifacts going to a common location I don't think is typically expected, them falling in a `target` or `build` file in the sub-project is typical.

Updating the path location is pretty straightforward; at first impression it does seem simpler compared to updating the build process to move the built file.